pkg/controllers/binding/common.go

```diff
@@ -252,3 +256,14 @@ func divideReplicasByJobCompletions(workload *unstructured.Unstructured, cluster
 
 	return targetClusters, nil
 }
+
+func hasReplicaSchedulingPolicy(appliedPlacement map[string]string) (bool, error) {
+	placement, err := helper.GetAppliedPlacement(appliedPlacement)
+	if err != nil {
+		return false, err
+	}
+	if placement == nil {
```

Comment:
When `placement` is nil,  it indicates that the scheduler has not scheduled the scheduler. In this case, the scheduling result is empty, so I think we don't need to return an error.

Reply:
OK, it may be better to add a judgement first. If `len(targetClusters) == 0`, function `ensureWork` would return. So if placement is nil here, which also indicates `len(targetClusters) != 0`, there must be something wrong here.